src/App.js

```diff
@@ -4,17 +4,21 @@ import timelineData from './data/timeline.json';
 
 import Timeline from './components/Timeline';
 
+
 class App extends Component {
+
   render() {
-    console.log(timelineData);
 
-    // Customize the code below
+    const dataArray = timelineData.events
+    const varHoldsTimeEvents = Timeline(dataArray); 
+
     return (
       <div className="App">
         <header className="App-header">
           <h1 className="App-title">Application title</h1>
         </header>
         <main className="App-main">
+          {varHoldsTimeEvents}
         </main>
```

Comment:
I like this variable name `varHoldsTimeEvents`, helps me remember that its prob a grouping of timeline events.